core/os/device/device.proto

```diff
@@ -175,8 +175,13 @@ message Configuration {
   PerfettoCapability perfetto_capability = 5;
 }
 
+message VulkanProfilingLayers {
+  bool cpu_timing = 1;
+}
```

Comment:
I would just put this into the `GPUProfiling` message below...

Reply:
I had wanted to keep this out, because the Vulkan profiling is complementary to the standard "GPU Profiling". It's going to be more than just GPU profiling as time goes on. (E.g. this layer actually does nothing GPU related)